### ISSUE TITLE
Feed parser utilty does not support Python 3

### ISSUE THREAD

**Issue by cccaballero** [web2py/web2py#1965]
gluon.contrib.feedparser is not Python 3.x compatible

**Comment by leonelcamara** [comment]
It's the latest feedparser release. There's a develop branch:

https://github.com/kurtmckee/feedparser/tree/develop/feedparser

You can try using that instead of the included with web2py version.

**Comment by cccaballero** [comment]
@leonelcamara yes, I am using it from pip right now and works ok (i think that uses 2to3 on install), I just want to point that the bundled version is broken with Python 3.x
